std/stdio.d

```diff
@@ -2466,12 +2466,6 @@ $(D Range) that locks the file and allows fast writing to it.
         _iobuf* handle_;     // the unshared version of fps
         int orientation_;
     public:
-        deprecated("accessing fps/handle/orientation directly can break LockingTextWriter integrity")
```

Comment:
git blame says this has been deprecated for ~1.5Years now, but no remove date was ever mentioned.

please do not get me wrong. This should go, but IMO we first sould say when.